ruff_lsp/server.py

```diff
@@ -1157,10 +1157,27 @@ async def _run_check_on_document(
     executable = _find_ruff_binary(settings, VERSION_REQUIREMENT_LINTER)
     argv: list[str] = CHECK_ARGS + list(extra_args)
 
+    skip_user_arg = False
     for arg in lint_args(settings):
+        if skip_user_arg:
+            skip_user_arg = False
+            continue
         if arg in UNSUPPORTED_CHECK_ARGS:
             log_to_output(f"Ignoring unsupported argument: {arg}")
         else:
+            if only:
```

Comment:
The check for unsupported seems like it could use `continue`?

Reply:
Are you referring to the other `if` branch? I might be misunderstanding but I don't think it does as it's a `if ... else ...`?

Reply:
Like

```
        if arg in UNSUPPORTED_CHECK_ARGS:
            log_to_output(f"Ignoring unsupported argument: {arg}")
            continue
       if only:
            ...
```

It doesn't look like there's any reason to use an `else` and indent all this code.

Reply:
Oh interesting! Yeah, this looks far more readable. I'll update the code, thanks!